forte/results.py

```diff
@@ -0,0 +1,20 @@
+class Results:
```

Comment:
I don't think we have a unit test file for this.

Reply:
Added.